clients/java/src/main/java/io/camunda/client/impl/search/SearchResponseMapper.java

```diff
@@ -53,4 +55,20 @@ public static SearchQueryResponse<ProcessInstance> toProcessInstanceSearchRespon
 
     return new SearchQueryResponseImpl<>(instances, page);
   }
+
+  public static SearchQueryResponse<UserTaskItem> toUserTaskSearchResponse(
+      final UserTaskSearchQueryResponse response) {
+    final SearchQueryPageResponse pageResponse = response.getPage();
+    final SearchResponsePage page = toSearchResponsePage(pageResponse);
+
+    return new SearchQueryResponseImpl<>(response.getItems(), page);
```

Comment:
❌ The return type of the query should be a client API class, not the REST model. Have a look at the process instance search above.